MedallionShell.Tests/MedallionShell.Tests.csproj

```diff
@@ -34,5 +35,14 @@
     <Reference Include="System.Management" />
     <Reference Include="System.ServiceModel" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.2'">
```

Comment:
According to https://www.nuget.org/packages/Microsoft.Bcl.AsyncInterfaces/8.0.0#dependencies-body-tab, 
`Microsoft.Bcl.AsyncInterfaces` 8.0.0 supports `net462`, `netstandard2.0`, and `netstandard2.1`. Since this project supports `net46` and `netcoreapp2.2`, I added this `Condition`.

Reply:
- [ ] Is this okay?

Reply:
Since this is the tests project, let’s just update the target frameworks version for .net framework to net472, which should eliminate the need for the condition.

We have to run the tests under some version of the framework; 472 is more likely to be used in the wild

Reply:
@madelson Actually, https://github.com/madelson/MedallionShell/compare/master...release-1.7#diff-d1151716ba27dda78ca149178b29bd40eeb3a804db93bf6378e4bbdb1ab9a6abR4 says 
> must be net46, not net472 to run Mono tests

Reply:
Let's stick with net46 for now, then